pages/project-details/[id]/index.js

```diff
@@ -71,6 +76,38 @@ export default function ProjectDetails({
     setEditImage(false);
   }
 
+  function addNewMaterialProject(id, newDimension) {
```

Comment:
does this `id` actually need to be an argument in that function or is it the same `id` you have in line 17?

Reply:
you right i didnt saw it